<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rechargeDialog</class>
 <widget class="QDialog" name="rechargeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>rechargeDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
border-image: url(:/masterDesktop/Resources/complaints_bg.png);
}
#btn_close{
border-image: url(:/masterDesktop/Resources/close.png);
}
#btn_close:pressed{
margin:1px;
}
#wgt_title{
background:#002E29;
border:1px solid #00947E;
}
#lbl_title{
font-family:&quot;微软雅黑&quot;;
font-size:18px;
color:#FFFFFF;
}
#wgt_tip, #wgt_content{
background:#0C2F2B;
border:1px solid #008272;
}</string>
     </property>
     <widget class="QPushButton" name="btn_close">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>28</y>
        <width>37</width>
        <height>37</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>37</width>
        <height>37</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>37</width>
        <height>37</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>82</y>
        <width>412</width>
        <height>507</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="wgt_title" native="true">
         <property name="minimumSize">
          <size>
           <width>410</width>
           <height>42</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>410</width>
           <height>42</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_title">
            <property name="text">
             <string>网费充值</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#scrollArea,#scrollAreaWidgetContents{
background:transparent;
border:none;
}
</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>410</width>
               <height>200</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>25</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="wgt_tip" native="true">
            <property name="minimumSize">
             <size>
              <width>410</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>410</width>
              <height>40</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>21</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lbl_activity">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#FFFFFF;</string>
               </property>
               <property name="text">
                <string>优惠活动</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_activityContent">
               <property name="styleSheet">
                <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:rgba(255,255,255,0.8);</string>
               </property>
               <property name="text">
                <string>全游电竞节，充100送100！</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wgt_content" native="true">
            <property name="minimumSize">
             <size>
              <width>410</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>410</width>
              <height>40</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>21</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#2EB4AF;</string>
               </property>
               <property name="text">
                <string>使用优惠券</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>207</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btn_select">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:rgba(255,255,255,0.7);
background:transparent;
}
QPushButton:hover{
color:#FEFEFE;
}
</string>
               </property>
               <property name="text">
                <string>选择优惠券   &gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_pay">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background:#2EB4AF;
border:none;
border-radius:2px;
font-family:&quot;微软雅黑&quot;;
font-size:16px;
color:#151515;
}
QPushButton:pressed{
margin:1px;
}</string>
              </property>
              <property name="text">
               <string>支付</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>76</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="masterDesktop.qrc"/>
 </resources>
 <connections/>
</ui>
